mungegithub/mungers/publish_scripts/util.sh

```diff
@@ -58,6 +63,9 @@ sync_repo() {
 
     local previousBranchSHA=$(git log --grep "Kubernetes-commit: ${previousKubeSHA}" --format='%H')
     local commits=$(git log --no-merges --format='%H' --reverse ${previousBranchSHA}..HEAD)
+    if [ "${new_repo}" = "true" ]; then
+        commits=$(git log --no-merges --format='%H' --reverse HEAD)
```

Comment:
This is really fine, but you could use `git rev-list` instead of `git log --format='%H'`.